examples/opensuse.yaml

```diff
@@ -3,7 +3,10 @@ images:
 # Hint: run `limactl prune` to invalidate the "Current" cache
 - location: https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-JeOS.x86_64-15.4-OpenStack-Cloud-Current.qcow2
   arch: "x86_64"
-# No aarch64 OpenStack build found :(
+  digest: "sha256:378c51ae8a064513410285ffe884fa455e6325c583861aebe11cff7291298195"
+- location: https://download.opensuse.org/distribution/leap/15.4/appliances/openSUSE-Leap-15.4-ARM-JeOS-efi.aarch64.qcow2
```

Comment:
JeOS is deprecated https://github.com/openSUSE/get-o-o/pull/123

Reply:
Yeah, I've asked internally to get a Minimal arm64 build with cloud-init support, but I have not received any notifications related to that.

But I think that would be a Tumbleweed build, not Leap.

Given that the amd64 build is also JeOS, I think using the JeOS build for arm64 is fine. I can ask for clarification on what the status is.